pkgs/applications/misc/plover/default.nix

```diff
@@ -1,37 +1,131 @@
-{ lib, fetchFromGitHub, python3Packages, wmctrl, qtbase, mkDerivationWith }:
+{
+  lib,
+  fetchFromGitHub,
+  # Plover is currently tested with python 3.10 and do not seems to work with more recent versions
+  # https://github.com/openstenoproject/plover/commits/a8ac3631bee1971eec2b41b74fbebdad4750291a
+  python310Packages,
+  qt5,
+  mkDerivationWith,
+}:
 
 {
-  stable = throw "plover.stable was removed because it used Python 2. Use plover.dev instead."; # added 2022-06-05
+  dev =
+    let
+      inherit (python310Packages)
+        appdirs
+        babel
+        buildPythonPackage
+        mock
+        pyqt5
+        pyserial
+        pytestCheckHook
+        setuptools
+        wcwidth
+        xlib
+        stdenv
+        ;
+      inherit (lib) licenses maintainers platforms;
+      plover_stroke = mkDerivationWith buildPythonPackage rec {
```

Comment:
After looking up what the heck `mkDerivationWith` is, I think we should move away from it and just use `qt5.wrapQtAppsHook` or something similar in a regular Python package (see #180841)

Reply:
I'll take a look at that later :+1:

Reply:
done